koa-basic-auth/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
That PR edits a lot of old packages. Older packages have `"strictNullChecks": false` because they were written for TS1.0 which didn't have null checks, so we need people to update older packages by adding `| null` and `| undefined` as appropriate, then setting that flag.